src/main/java/blusunrize/immersiveengineering/common/util/compat/crafttweaker/CrTIngredientUtil.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * BluSunrize
+ * Copyright (c) 2020
+ *
+ * This code is licensed under "Blu's License of Common Sense"
+ * Details can be found in the license file in the root folder of this project
+ */
+package blusunrize.immersiveengineering.common.util.compat.crafttweaker;
+
+import blusunrize.immersiveengineering.api.crafting.FluidTagInput;
+import blusunrize.immersiveengineering.api.crafting.IngredientWithSize;
+import blusunrize.immersiveengineering.api.crafting.StackWithChance;
+import com.blamejared.crafttweaker.api.item.IIngredient;
+import com.blamejared.crafttweaker.api.item.IItemStack;
+import com.blamejared.crafttweaker.impl.item.MCWeightedItemStack;
+import com.blamejared.crafttweaker.impl.tag.MCTag;
+import net.minecraft.fluid.Fluid;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.crafting.Ingredient;
+import net.minecraft.tags.ITag;
+import net.minecraft.util.NonNullList;
+import net.minecraft.util.math.MathHelper;
+
+public class CrTIngredientUtil
+{
+
+	private CrTIngredientUtil()
+	{
+	}
+
+	/**
+	 * So far, only IItemStack supports setting an amount.
+	 * So we can at least check for that, I guess?
+	 */
+	public static IngredientWithSize getIngredientWithSize(IIngredient crafttweakerIngredient)
```

Comment:
Would it make sense to have count somehow be applied within CraftTweaker to IIngredients? I also use count on Tags and such in Transport, and definitely see it as a useful addition (And I think it is something that is useful? Only having ItemStacks be able to have count for the integration with IE seems slightly less useful?)